Dockerfile.enhanced

```diff
@@ -23,7 +23,7 @@ RUN cd ./modules/oas-page-builder \
 FROM ubuntu:20.04
 ARG WORK_DIRECTORY=/home/docsworker-xlarge
 ARG SNOOTY_PARSER_VERSION=0.15.0
-ARG SNOOTY_FRONTEND_VERSION=0.15.1.1b
+ARG SNOOTY_FRONTEND_VERSION=DOP-3998
```

Comment:
TODO: revert this change. testing with front end changes in feature branch stack